### PR TITLE
Adds configurable and mappable hidden fields.

### PR DESCRIPTION
**Pull request by haogatyp**

### CODE REVIEW

**Comment by claussni** [comment]
I can see what is happening here: Hidden fields can be defined with default values and XML mappings to ensure certain elements in the resulting XML.  Is this correct? Or is there another use case?

File: Resources/Private/Language/de.locallang_db.xlf [hunk 1]
```diff
@@ -405,6 +405,10 @@
 			<source>checkbox</source>
 			<target>Checkbox</target>
 		</trans-unit>
+		<trans-unit id="tx_dpf_domain_model_metadataobject.input_type.hidden" approved="yes">
+			<source>hidden</source>
+			<target>Hidden</target>
```
Comment (claussni) [inline Resources/Private/Language/de.locallang_db.xlf]: Translation should be "Versteckt" not "Hidden".

Comment (haogatyp) [reply Resources/Private/Language/de.locallang_db.xlf]: The translation has been adjusted accordingly.

Comment (haogatyp) [reply Resources/Private/Language/de.locallang_db.xlf]: The use case is exactly as described above.

**Comment by haogatyp** [comment]
The use case is exactly as described above.

**Comment by haogatyp** [comment]
The translation was adjusted as proposed.

**Pull request merged**